### DISCUSSION TITLE
Put Entities and ValueObjects into seperate NuGet

### DISCUSSION THREAD

**Discussion by cmenzi** [vkhorikov/CSharpFunctionalExtensions#532] [General]
I like the functional classes (Maybe and Result) very much. They really great using in DDD, but I have my own Aggregates, Entities and ValueObjects. 

Therefore it would be great if the `CSharpFunctionalExtensions` would only contain the functional stuff and keep the DDD concepts seperated.

I mean, the classes could be just copied into a project, but it would be great to consume a nuget.

**Comment by vkhorikov** [comment]
Yeah, there's an issue for splitting all the classes into their own separate NuGet packages, but this is quite a bit of work. As a temporary measure, you can just copy Maybe and Result with their extension to your project, so that Entity and ValueObject from here doesn't overlap with your own implementations.
